src/nft/utils/fetchPrice.ts

```diff
@@ -20,5 +20,5 @@ export const fetchPrice = async (currency: Currency = Currency.ETH): Promise<num
 
 export function useUsdPrice(asset: GenieAsset): string {
   const { data: fetchedPriceData } = useQuery(['fetchPrice', {}], () => fetchPrice(), {})
-  return fetchedPriceData ? (parseFloat(formatEther(asset.priceInfo.ETHPrice)) * fetchedPriceData).toString() : ''
+  return fetchedPriceData ? (parseFloat(formatEther(asset?.priceInfo?.ETHPrice)) * fetchedPriceData).toString() : ''
```

Comment:
when an asset isn't listed for sale this causes an error check if the asset has a price first / is available